providers/flagd/pkg/provider.go

```diff
@@ -472,6 +472,10 @@ func (p *Provider) handleEvents(ctx context.Context) error {
 				WithoutCache()(p)
 			}
 			return err
+		case <-ctx.Done():
+			log.Info("Stop event handling with context done.")
```

Comment:
```suggestion
			log.Info("stop event handling with context done")
```
Usual practice is lowercase without period